src/System.Management.Automation/engine/hostifaces/History.cs

```diff
@@ -36,11 +34,11 @@ internal HistoryInfo(long pipelineId, string cmdline, PipelineState status, Date
         {
             Dbg.Assert(cmdline != null, "caller should validate the parameter");
             _pipelineId = pipelineId;
-            _cmdline = cmdline;
-            _status = status;
-            _startTime = startTime;
-            _endTime = endTime;
-            _cleared = false;
+            CommandLine = cmdline;
```

Comment:
@SteveL-MSFT @PaulHigin Does this not break remoting?

Reply:
There's a specific [list of types](https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/remoting/common/WireDataFormat/RemoteHostEncoder.cs#L41) and HistoryInfo isn't in that list